src/components/DateTemperature/index.js

```diff
@@ -0,0 +1,56 @@
+import "macro-css";
+import { monthNames } from "../../utilities/mockData"
+import { MainBlockDay } from "./index1";
+
+
+
+export const MainBlock = ({ data, date }) => {
+	const temp = (n) => {
+		return (n > 0 ? "+" + Math.floor(n) : "-" + Math.floor(n))
+	}
+	const d = new Date();
+	if (!data.list.length) return null
+	return (
```

Comment:
will be better use a  ternary operator. return !data.list.length ? null : <div> .... </div>